apps/directory/src/components/Footer.vue

```diff
@@ -0,0 +1,145 @@
+            </a>
+            <a
+              v-if="isFooterFollowUsVisible"
+              href="https://www.bbmri-eric.eu/bbmri-eric/bbmri-eric-podcast/"
+              class="link-dark ms-2"
+            >
+              <i class="bi bi-mic-fill" />
+            </a>
+            <div class="mt-2 mb-2 mb-md-0">
+              <a
+                href="https://https://github.com/molgenis/molgenis-emx2"
+                class="text-primary-text"
+              >
+                <i class="bi bi-github" />
+                GitHub UI
+              </a>
+              <a
+                href="https://https://github.com/molgenis/molgenis-emx2"
+                class="text-primary-text ps-2"
+              >
+                <i class="bi bi-github" />
+                GitHub Application
+              </a>
+            </div>
+
+            <div v-if="isFooterFollowUsVisible" class="mt-2 mb-2 mb-md-0">
+              <a href="/api/swagger-ui/index.html" class="text-primary-text">
+                <i class="bi bi-braces-asterisk" />
+                API
+              </a>
+              <a
+                href="https://status.bbmri-eric.eu/"
+                class="text-primary-text ps-2"
+              >
+                <i class="bi bi-check-circle" />
+                BBMRI-ERIC Status page
+              </a>
+            </div>
+          </div>
+
+          <div class="col text-center">
+            <a
+              class="link-dark text-primary-text"
+              href="https://www.bbmri-eric.eu/wp-content/uploads/BBMRI-ERIC_work-program_2022-2024_DIGITAL.pdf"
+            >
+              <img
+                width="22"
+                height="22"
+                src="/img/work-programme.png"
+                alt="Work programme image"
+              />
+              Work Programme
+            </a>
+            <div v-if="isFooterFollowUsVisible" class="ms-md-3 mt-2">
+              <a
+                role="button"
+                class="btn btn-light ms-md-5 text-primary-text"
+                href="https://www.bbmri-eric.eu/news-event/"
+              >
+                Subscribe To Our Newsletter
+              </a>
+            </div>
+            <div v-if="isFooterFollowUsVisible" class="ms-md-2 mt-2">
+              <a
+                class="link-dark text-primary-text me-5"
+                href="https://www.bbmri-eric.eu/wp-content/uploads/AoM_10_8_Access-Policy_FINAL_EU.pdf"
+              >
+                Privacy Policy
+              </a>
+            </div>
+          </div>
+          <div>
+            <div class="row mt-2">
+              <div class="col text-center">
+                <p>&copy; 2024 BBMRI-ERIC</p>
+              </div>
+              <div class="col text-center text-light ms-4"></div>
+              <div class="col text-center ms-5">
+                Need help?
+                <a href="mailto:negotiator@helpdesk.bbmri-eric.eu">Contact us</a
+                >.
```

Comment:
broken up tag

Reply:
make this
```
<a ....> 
 content
</a>.
 ```

(I know the formatter allows the current state, which is a bit annoying)